content/tutorials/create-users-and-teams.md

```diff
@@ -12,40 +12,44 @@ draft: true
 
 {{% tutorials/step duration="1" title="Introduction" %}}
 
-Create users and organize them into teams. Manage resource access for each user and team through roles and folders.
+This tutorial is useful for admins and others who want to learn how to manage users. In this tutorial, you'll add multiple users, organize them into teams, and make sure they're only able to access the resources they need.
 
-### What you'll learn
 
-- How to add users
-- How to assign users to teams
-- How to manage dashboard access using folders
-- How to override access for individual dashboards
+### You'll learn to
 
-### What you'll build
+- Add users.
+- Assign users to teams.
+- Manage dashboard access using folders.
+- Override access for individual dashboards.
+
+### Scenario
 
 _Graphona_, a fictional telemarketing company, has asked you to configure Grafana for their teams.
 
-In this tutorial, you'll add multiple users, organize them into teams, and make sure they're only able to access the resources they need.
+### You will
+* Create users and organize them into teams. 
+* Manage resource access for each user and team through roles and folders.
 
-### What you'll need
+### Prerequisites
 
 - Grafana 7.0
+((Does it matter if it is installed, running on Docker, or accessed some other way?))
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Add users" %}}
 
-In Grafana, all users are granted an _organization role_ that determines what resources the user can access.
+In Grafana, all users are granted an _organization role_ that determines what resources they can access.
 
 There are three types of organization roles in Grafana:
 
-- **Admin**, for managing data sources, teams, and users within an organization.
-- **Editor**, for creating and editing dashboards.
-- **Viewer**, for viewing dashboards.
+- **Admin -** For managing data sources, teams, and users within an organization.
+- **Editor -** For creating and editing dashboards.
+- **Viewer -** For viewing dashboards.
 
 
-> **Note**: You can also configure Grafana to allow [anonymous access](https://grafana.com/docs/grafana/latest/auth/overview/#anonymous-authentication), to make dashboards available even to those who don't have a Grafana user.
+> **Note**: You can also configure Grafana to allow [anonymous access](https://grafana.com/docs/grafana/latest/auth/overview/#anonymous-authentication), to make dashboards available even to those who don't have a Grafana user account. ((Do we want to point to play.grafana.com as an example?))
 
-### Exercise
+### Exercise: Add a user
```

Comment:
I'm using the Exercise heading to set the context and explain what the reader will build. The way I see it, an exercise can contain multiple tasks.

Reply:
Discussed, I agree. Let's keep the exercise and procedures headings separate.